### PR TITLE
aio-interface: adjust design for nc30

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/all-in-one/issues/4955

### CODE REVIEW

**Comment by docjyJ** [comment]
Hi, are you open to a proposal?

**Comment by szaimen** [comment]
> Hi, are you open to a proposal?

Hi @docjyJ, in general yes. :) What are you proposing?

**Comment by docjyJ** [comment]
I pushed it, feel free to delete the commit.

It's just the blue border effect of the classic aio + manage paragraphs in the flex column.

See 00f6789

**Comment by szaimen** [comment]
Ah, I see... I currently cannot test this. Because I am on my phone... Can you post a screenshot?

**Comment by docjyJ** [comment]
_Without the title in the header ^^'
I'm working on stalwart I don't have the all-in-one project open_

https://github.com/nextcloud/all-in-one/pull/5016#issuecomment-2241706188

**Comment by docjyJ** [comment]
> _Without the title in the header ^^' I'm working on stalwart I don't have the all-in-one project open_
> 
> [#5016 (comment)](https://github.com/nextcloud/all-in-one/pull/5016#issuecomment-2241706188)

_and without the dividers_

File: php/public/style.css [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
     outline: none;
 }
 
 a.button:focus,
 input[type="submit"]:focus {
-    color: black;
-    border: 2px solid black;
+.button:focus {
+    border: 1px solid black;
 }
```
Comment (docjyJ) [inline php/public/style.css:45]: The `.button:focus` selector is redundant to `a.button:focus`.
I'm not sure the `.button:focus` selector is useful...

File: Containers/mastercontainer/Dockerfile [hunk 2]
```diff
@@ -64,7 +64,7 @@ RUN set -ex; \
     wget https://getcomposer.org/installer -O - | php -- --install-dir=/usr/local/bin --filename=composer; \
     chmod +x /usr/local/bin/composer; \
     cd /var/www/docker-aio; \
-    git clone https://github.com/nextcloud-releases/all-in-one.git --depth 1 .; \
+    git clone https://github.com/nextcloud/all-in-one.git --depth 1 --branch enn/4955/adjust-design .; \
```
Comment (szaimen) [inline Containers/mastercontainer/Dockerfile:67]: TODO: revert this later on

**Comment by szaimen** [comment]
Thanks @docjyJ I used some of your proposal and fixed some details :)

**Comment by szaimen** [comment]
This is now released with 9.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Pull request merged**